cranelift/codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -2187,12 +2189,14 @@ fn define_entity_ref(
         is_pic,
     );
 
-    // Stack addresses.
-    //
-    // TODO: Add encoding rules for stack_load and stack_store, so that they
-    // don't get legalized to stack_addr + load/store.
-    e.enc32(stack_addr.bind(I32), rec_spaddr4_id.opcodes(&LEA));
-    e.enc64(stack_addr.bind(I64), rec_spaddr8_id.opcodes(&LEA).rex().w());
+    // Stack accesses.
+    e.enc32(stack_addr.bind(I32), rec_spaddr_ld_id.opcodes(&LEA));
+    e.enc64(
+        stack_addr.bind(I64),
+        rec_spaddr_ld_id.opcodes(&LEA).rex().w(),
+    );
+    e.enc_i32_i64_explicit_rex(stack_load, rec_spaddr_ld_id.opcodes(&MOV_LOAD));
```

Comment:
I see _why_ this encoding rule is correct, but at a glance this reads like a bug - I'd at first expected the recipe to be `rec_spld_id` for `recipe sp-relative load yielding an integer`, and mirroring the store.

What do you think of using `rec_spld_id` as the name here? I think that fits a bit better with the `LEA` encodings since it reads as "load the effective address of what would be a stack load".